src/layouts/_macros.html

```diff
@@ -12,10 +12,10 @@ <h1 class="subdued-links" id="{{ projectInfo.anchor }}">{{ projectInfo.title }}<
     {% set project = projectInfo.projects[0] %}
     <div class="panel panel-gray">
       <div class="column-8 tablet-column-4 phone-column-5">
-        <img src="{{ project.image }}" />
+        <a href="{{ project.url }}" alt="{{ project.title }}"><img src="{{ project.image }}" /></a>
       </div>
       <div class="column-12 tablet-column-6 phone-column-5">
-        <h4>{{ project.title }}</h4>
+        <h4><a href="{{ project.url }}">{{ project.title }}</a></h4>
```

Comment:
initially i thought that the the only kind of link that would make sense for the case studies would be more information about the actual implementation.

but i was probably letting perfect be the enemy of the good.